test/integration/test_config_defaults.py

```diff
@@ -126,6 +127,7 @@
     "shed_tool_config_file": "managed_config_dir",
     "shed_tool_data_path": "tool_data_path",
     "shed_tool_data_table_config": "managed_config_dir",
+    "short_term_storage_directory": "cache_dir",
```

Comment:
what is the difference between `short_term_storage_dir` and `short_term_storage_directory`

Reply:
Dropped this config test change - I think this was a no-op.